### PR TITLE
added conda-install-option and badges to readme

### LINKED ISSUE #14: Add a conda installation option using conda-forge channel
I have already started this one. Will push a PR once the `conda` installation option is available.

See: [Adding doubtlab from PyPI](https://github.com/conda-forge/staged-recipes/pull/16940) to `conda-forge` channel.

@koaning As the primary maintainer of this repo, would you like to be listed as one of the maintainers of `doubtlab` on `conda-forge` channel? Please let me know, I will add your name as another maintainer of `conda-forge/doubtlab-feedstock`, once it is accepted.

### PR DESCRIPTION
**Pull request by sugatoray**
This closes #14: `doubtlab` can now be installed with **conda** from `conda-forge` channel. 

- [x] Created [conda-forge/doubtlab-feedstock](https://github.com/conda-forge/doubtlab-feedstock) to make `doubtlab` available on conda-forge channel. 
- [x] Added conda install option to readme.
- [x] Added the following badges to readme.

[GitHub - License][#github-license]
[PyPI - Python Version][#pypi-package]
[PyPI - Package Version][#pypi-package]
[PyPI - Downloads][#pypi-package]
[Conda - Platform][#conda-forge-package]
[Conda (channel only)][#conda-forge-package]
[Docs - GitHub.io][#docs-package]

[#github-license]: https://github.com/koaning/doubtlab/blob/main/LICENSE
[#pypi-package]: https://pypi.org/project/doubtlab/
[#conda-forge-package]: https://anaconda.org/conda-forge/doubtlab
[#docs-package]: https://koaning.github.io/doubtlab/

### CODE REVIEW

**Comment by sugatoray** [comment]
cc: @koaning

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+<!--- BADGES: START --->
+[![GitHub - License](https://img.shields.io/github/license/koaning/doubtlab?logo=github&style=flat&color=green)][#github-license]
+[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/doubtlab?logo=pypi&style=flat&color=blue)][#pypi-package]
+[![PyPI - Package Version](https://img.shields.io/pypi/v/doubtlab?logo=pypi&style=flat&color=orange)][#pypi-package]
+[![Conda - Platform](https://img.shields.io/conda/pn/conda-forge/doubtlab?logo=anaconda&style=flat)][#conda-forge-package]
```
Comment (koaning) [inline README.md:5]: What does the conda platform indicate exactly?

**Comment by koaning** [comment]
I'd like to understand one of the shields better before I hit merge. Also, could you remove the comments from the readme?

File: README.md (hunk 1 above)
Comment (sugatoray) [reply README.md:5]: When a package is made available on a conda channel, you could make it available selectively for any or a combination of operating systems: windows, mac os, linux/unix. Typically, if the library does not have some os-specific limitations, the preferred way of packaging it on conda-forge channel is to make it architecture agnostic (`noarch`).

The conda `platform` basically specifies what platforms (os: win/mac/unix) it is available for. In our case, it is `noarch` (it is architecture-agnostic).

image

For instance, `boto3` library is not packaged as `noarch` ALONE, and it is possible to have different versions of the same package available for different platforms.

image

**Comment by sugatoray** [comment]
> I'd like to understand one of the shields better before I hit merge. Also, could you remove the comments from the readme?

Done.

**Comment by koaning** [comment]
LGTM! Thanks!

**Pull request merged**